src/runtime/audience-action-flow.ts

```diff
@@ -296,7 +296,16 @@ export class AudienceActionFlow {
       this.storage_.storeEvent(StorageKeys.SURVEY_DATA_TRANSFER_FAILED);
     }
     const surveyDataTransferResponse = new SurveyDataTransferResponse();
-    surveyDataTransferResponse.setSuccess(dataTransferSuccess);
+    const isPpsEligible = request.getStorePpsInLocalStorage();
+
+    if (isPpsEligible) {
+      const configurePpsSuccess = await this.configureAnswerPpsData(request);
+      surveyDataTransferResponse.setSuccess(
+        configurePpsSuccess && dataTransferSuccess
```

Comment:
hmm ok that makes sense. i can put a client log in order to handle this which will be more useful but since it doesn't affect data transfer it makes sense that iterating on both tese variables is not wise.